spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -1727,6 +1727,17 @@ def setup_firefox_with_linux
       end
     end
 
+    it 'disables the add new iso datastore button when no EMSes are available' do
+      allow(ManageIQ::Providers::Redhat::InfraManager)
+        .to(receive(:any_without_datastores?))
+        .and_return(false)
+
+      @layout = "pxe"
+      @id = "iso_datastore_new"
+
+      expect(subject).to be_truthy
```

Comment:
I see `subject` defined [here](https://github.com/ManageIQ/manageiq/pull/10046/files#diff-7ae614daa2bb6b8fb8409c6f36995139R1662), but I can't say whether or not being truthy is a good test.